iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -465,6 +471,11 @@ public ModuleRegistrationResponse processRegistrationRequest(ModuleRegistrationR
                         .setIsChecked(true)
                         .setDescription("If enabled, mParticle will attempt to coerce string attributes into scalar types (integer, boolean, and float).")
         );
+        eventSettings.add(
+                new TextSetting(SETTING_USER_ID_FIELD, "User ID field")
```

Comment:
I'm not sure how to make a drop-down out of this, like i've seen in other partner integrations..

Reply:
Looks like this is something mParticle has to do manually